docs/src/content/docs/sdk.mdx

```diff
@@ -61,7 +61,7 @@ The core of the SDK is the `UltravoxSession`. The session is used to join and le
 </Tabs>
 
 ### Methods
-The `UltravoxSession` contains methods for joining/leaving a call, for sending text messages to the model and for muting the microphone/speaker.
+The `UltravoxSession` contains methods for joining/leaving a call, for sending text messages to the model, for registering client tools, and for muting the microphone/speaker.
```

Comment:
Also setting the output medium, but does it really make sense to list all the method of an open-source object?

Reply:
added. i don't think we can assume that everyone will want to look at the code for the SDK to figure out what the methods are or how they work so i think it's worth it to list out the methods here for now.

Reply:
Just my two cents, but my thought was that:

1. It makes sense to describe the core flows to get people in the door.
2. It makes sense to describe how to use vertical features (e.g. client-implemented tools) in a feature-centric way to help folks take advantage of those.
3. It doesn't make sense to me to describe every SDK method with no other context. Once someone is developing, they almost certainly have a language server (typically bundled with their IDE) that can tell them what methods are available on an object and pull in the docstring for those methods. That source is always going to be up to date and reflect both their language and SDK version. (And the few people who don't have a language server running can view the SDK source.)

Reply:
yeah i think you are correct for the most part. i think there might still be a case for listing methods/capabilities in the docs for folks just checking things out. and if we can get the javadoc-type stuff that you started to be fleshed out in all the SDKs then yeah i agree. thx for the input!